docs/Tasks/installing-an-extension.md

```diff
@@ -1,62 +1,106 @@
-  resolvedBundleResource: quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6
-```
-
-The status condition type `Installed`:`true` indicates that the extension was installed successfully. We can confirm this by looking at the workloads that were created as a result of this extension installation: 
-
-```bash 
-$ kubectl get namespaces | grep argocd 
-argocd-operator-system       Active   4m17s
-
-$ kubectl get pods -n argocd-operator-system 
-NAME                                                 READY   STATUS    RESTARTS   AGE
-argocd-operator-controller-manager-bb496c545-ljbbr   2/2     Running   0          4m32s
-```
+# Installing an extension from a catalog
+
+In Operator Lifecycle Manager (OLM) 1.0, Kubernetes extensions are scoped to the cluster.
+After you add a catalog to your cluster, you can install an extension by creating a custom resource (CR) and applying it.
+
+!!! important
+
+    Currently, extensions that use webhooks or target a single or specified set of namespaces cannot be installed.
+    Extensions must not include webhooks and must use the `AllNamespaces` install mode.
+
+
+## Prerequisites
+
+* The `jq` CLI tool is installed.
+* You have added a catalog to your cluster.
+
+## Procedure
+
+1. Create a CR for the Kubernetes extension you want to install:
+
+    ``` yaml title="Example CR"
+    apiVersion: olm.operatorframework.io/v1alpha1
+    kind: ClusterExtension
+    metadata:
+      name: <extension_name>
+    spec:
+      packageName: <package_name>
+      channel: <channel>
+      version: "<version>"
+    ```
+
+     `extension_name`
+     : Specifies a custom name for the Kubernetes extension you want to install, such as `my-camel-k`.
+
+     `package_name`
+     : Specifies the name of the package you want to install, such as `camel-k`.
+
+     `channel`
+     : Optional: Specifies the extension's channel, such as `stable` or `candidate`.
+
+     `version`
+     : Optional: Specifies the version or version range you want installed, such as `1.3.1` or `"<2"`.
+     If you use a comparison string to define a version range, the string must be surrounded by double quotes (`"`).
+
+    !!! warning
+        Currently, the following limitations affect the installation of extensions:
+
+        * If mulitple catalogs are added to a cluster, you cannot specify a catalog when you install an extension.
+        * OLM 1.0 requires that all of the extensions have unique bundle and package names for dependency resolution.
+
+        As a result, if two catalogs have an extension with the same name, the installation might fail or lead to an unintended outcome.
+        For example, the first extension that matches might install successfully and finish without searching for a match in the second catalog.
+
+2. Apply the CR the cluster:
+
+    ``` terminal
+    $ kubectl apply -f <cr_name>.yaml
+    ```
+
+    ??? success
```

Comment:
Why is there `???` ?

Reply:
This adds a toggle to hide the output results once the MD is rendered. You can see it here in the rendered preview, step 2: https://deploy-preview-700--olmv1.netlify.app/tasks/installing-an-extension/